framework/features/PulseFeature.h

```diff
@@ -0,0 +1,66 @@
+#ifndef PULSEFEATURE_H
+#define PULSEFEATURE_H
+
+#include "Feature.h"
+
+template<const char* const name, uint16_t gpio, bool invert = false, uint16_t damper = 0, uint16_t max_duration = 0>
+class PulseFeature : public Feature<name> {
+
+protected:
+    using Feature<name>::LOG;
+
+public:
+    PulseFeature(Device* device) :
+            Feature<name>(device),
+            lastChange(RTC.getRtcSeconds()) {
+        pinMode(gpio, OUTPUT);
+        digitalWrite(gpio, invert);
+        this->switchBackTimer.initializeMs(100, TimerDelegate(&PulseFeature::switchOff, this));
+        this->registerSubscription("pulse", Device::MessageCallback(&PulseFeature::onMessageReceived, this));
+
+        LOG.log("Initialized");
+    }
+
+    void doPulse(const int ms) {
```

Comment:
Should we check for `duration <= max_duration` here? Or maybe just use `min(duration, max_duration)`?